File: Back/src/ProEventos.Domain/Evento.cs
﻿using ProEventos.Domain;
using ProEventos.Domain.Identity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEvento.Domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }
        public string Tema { get; set; }
        public int QuantidadePessoas { get; set; }
        public string ImagemURL { get; set; }
        public  string Telefone{ get; set; }
        public string Email{ get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial> RedeSociais{ get; set; }
        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}

File: Front/ProEventos-App/src/app/models/Evento.ts
import { User } from "./Identity/User";
import { Lote } from "./Lote";
import { Palestrante } from "./Palestrante";
import { RedeSocial } from "./RedeSocial";

export interface Evento {
  id: number;
  local: string;
  dataEvento: Date;
  tema: string;
  quantidadePessoas: number;
  imagemURL: string;
  telefone: string;
  email: string;
  lotes: Lote[];
  redeSociai: RedeSocial[];
  palestrantesEventos: Palestrante[];
  userDto: User;
}
